resources/keywords/deploy_application.resource

```diff
@@ -96,3 +95,26 @@ Load Data To Mongo Application
     ...  ELSE
     ...    Log  Connection Failed! Check IP Configuration For the Mongo DB Instance on ISV   WARN
 
+Open Mongo Application
+    ${linkName}=    Catenate    SEPARATOR=-     ${appname}      ${newProject}
+    ${appLink}=     Get Quarkus Application URL   ${OCP_CONSOLE_URL}      ${linkName}
+    Execute Javascript      window.open("${appLink}")
+    Switch Window   url:${appLink}
+
+Open Application
+    Element Should Be Visible    ${lnkAppRoute}
+    ${appLink}=     Get Element Attribute    ${lnkAppRoute}    href
+    Click Element    ${lnkAppRoute}
+    Switch Window   url:${appLink}
+
+Load Data To Application
+    ${appStatus}=    Run Keyword And Return Status    Wait Until Element Is Visible    ${hdrCRUDApp}
+    Run Keyword If    not ${appStatus}
+    ...    Reload Page
+    Run Keyword And Return Status    Wait Until Element Is Visible    ${hdrCRUDApp}
```

Comment:
We can have only Wait Until Keyword instead of Run Keyword and Return Status, as we are not utilizing the status anywhere